src/ert/storage/local_storage.py

```diff
@@ -252,17 +259,20 @@ def create_experiment(
         parameters: Optional[List[ParameterConfig]] = None,
         responses: Optional[List[ResponseConfig]] = None,
         observations: Optional[Dict[str, xr.Dataset]] = None,
+        name: Optional[str] = None,
```

Comment:
Should this be defaulted? In the migration it is added explicitly, instead of being None, indicating that maybe it is required